src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/SingleShot.cs

```diff
@@ -103,7 +117,14 @@ private void CreateSingleShot(string modelAbsPath, TensorsInfo inTensorInfo, Ten
             input_info = inTensorInfo.GetTensorsInfoHandle();
             output_info = outTensorInfo.GetTensorsInfoHandle();
 
-            ret = Interop.SingleShot.OpenSingle(out _handle, modelAbsPath, input_info, output_info, NNFWType.Any, HWType.Any);
+            try
+            {
+                ret = Interop.SingleShot.OpenSingle(out _handle, modelAbsPath, input_info, output_info, NNFWType.Any, HWType.Any);
+            }
+            catch (DllNotFoundException e)
```

Comment:
DllNotFound is already checked in CheckNNStreamerSupport()

Reply:
ACK!